ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -317,6 +553,121 @@ export default {
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '7974', capture: false }),
+      suppressSeconds: 1,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Reflect Orbs',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Queen Fiery Portent',
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '5A21' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '5A21' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '5A21' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '5A21' }),
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - 2.5,
+      durationSeconds: 5.5,
+      response: Responses.stopEverything('alarm'),
+    },
+    {
+      id: 'DelubrumSav Queen Icy Portent',
+      // Assuming you need to move for 3 seconds (duration of Pyretic from Fiery Portent)
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '5A22' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '5A22' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '5A22' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '5A22' }),
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - 2.5,
+      durationSeconds: 5.5,
+      response: Responses.moveAround('alert'),
+    },
+    {
+      id: 'DelubrumSav Queen Guard AoEs',
+      // 5A16 from Queen's Warrior
+      // 5A08 from Queen's Knight
+      // 5A35 from Queen's Gunner
+      // 5A23 from Queen's Soldier
+      // These happen in sets:
+      // Set 1 Double AoE, 3 seconds later Double AoE
+      // Set 2 5 seconds later, Double AoE, 3 seconds later Double AoE, 3 seconds later AoE + Bleed
+      // Set 3 1.3 seconds later, Single AoEs every 3 seconds all while bleed from set 2 persists
+      netRegex: NetRegexes.startsUsing({ source: ['Queen\'s Warrior', 'Queen\'s Knight', 'Queen\'s Gunner', 'Queen\'s Soldier'], id: ['5A16', '5A08', '5A35', '5A23'], capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: ['Kriegerin Der Königin', 'Ritter Der Königin', 'Schütze Der Königin', 'Soldat Der Königin'], id: ['5A16', '5A08', '5A35', '5A23'], capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: ['Guerrière De La Reine', 'Chevalier De La Reine', 'Fusilier De La Reine', 'Soldat De La Reine'], id: ['5A16', '5A08', '5A35', '5A23'], capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: ['クイーンズ・ウォリアー', 'クイーンズ・ナイト', 'クイーンズ・ガンナー', 'クイーンズ・ソルジャー'], id: ['5A16', '5A08', '5A35', '5A23'], capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      // Only call out the beginning of a set of two casts
+      suppressSeconds: 5,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Multiple Guards\' AOEs',
```

Comment:
```suggestion
          en: 'Multiple Big AOEs',
```

This is just a tiny nitpick, but I think the word "Guard" isn't doing much here.  Maybe "Big" as I wrote or just "Multiple AOEs"?

Reply:
To avoid confusion between the other Big AoE callouts, I have shortened it to Multiple AoEs. I was also thinking it could be shortened to Guards' AoEs, but Multiple requires less thought about it.